plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -1188,6 +1197,32 @@ protected void verifyColumnName(DatabaseMetaData databaseMetadata, String column
         }
     }
 
+    @Override
+    public List<JdbcColumnHandle> getPrimaryKeys(ConnectorSession session, RemoteTableName remoteTableName)
```

Comment:
Why we cannot use `ctid`?

Reply:
The `ctid` value might be changed after the update.
see the blow logs, that I log before and after the update in the `testUpdateRowConcurrently`.
```
2024-11-28T20:36:22.057-0600	INFO	pool-104-thread-4	stdout	ctid :(0,1)
2024-11-28T20:36:22.057-0600	INFO	pool-104-thread-1	stdout	ctid :(0,1)
2024-11-28T20:36:22.057-0600	INFO	pool-104-thread-3	stdout	ctid :(0,1)
2024-11-28T20:36:22.057-0600	INFO	pool-104-thread-2	stdout	ctid :(0,1)
2024-11-28T20:36:22.380-0600	INFO	pool-104-thread-4	stdout	ctid :(0,5)
2024-11-28T20:36:22.381-0600	INFO	pool-104-thread-1	stdout	ctid :(0,5)
2024-11-28T20:36:22.383-0600	INFO	pool-104-thread-2	stdout	ctid :(0,5)
2024-11-28T20:36:22.383-0600	INFO	pool-104-thread-3	stdout	ctid :(0,5)
```